Source/Extensions/Blazorise.TreeView/Internal/_TreeViewNode.razor.cs

```diff
@@ -21,6 +21,16 @@ protected override async Task OnInitializedAsync()
         {
             await AutoExpandNodes();
         }
+        else if ( ExpandedNodes?.Count > 0 )
+        {
+            foreach ( var nodeState in NodeStates ?? Enumerable.Empty<TreeViewNodeState<TNode>>() )
+            {
+                if ( nodeState.HasChildren && ExpandedNodes?.Contains( nodeState.Node ) == true )
```

Comment:
Can replace `ExpandedNodes?.Contains( nodeState.Node ) == true` with `ExpandedNodes.Contains( nodeState.Node )` if you want to bother, as you've null checked above.